Workbooks/Azure Private MEC/PMEC Site Overview/PMEC Site Overview.workbook

```diff
@@ -1279,6 +1279,22 @@
             },
             "showPin": true,
             "name": "controlplaneerrors"
+          },
```

Comment:
In terms of placement, we should have the alerts show up just below the "Packet Core Status" section so it's easily visible on the page if there are alerts.